source/_components/geofency.md

```diff
@@ -17,28 +17,4 @@ This component sets up integration with [Geofency](http://www.geofency.com/). Ge
 
 Enabling this component will automatically enable the [Geofency Device Tracker](/components/device_tracker.geofency/).
 
-To integrate Geofency in Home Assistant, add the following section to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-geofency:
-```
-
-{% configuration %}
-mobile_beacons:
```

Comment:
This optional part is still needed. We have no way to add mobile_beacons from the UI.